Helm/opensearch/templates/statefulset.yaml

```
@@ -336,7 +336,7 @@ spec:
       {{- if eq .Values.roles.master "true" }}
       # This sidecar will prevent slow master re-election
       - name: opensearch-master-graceful-termination-handler
-        image: "{{ .Values.image }}:{{ .Values.imageTag }}"
+        image: "{{ .Values.image }}:{{ .Values.imageTag | default .Chart.AppVersion }}"
```

Possibly pedantic, but the correct syntax is this:

```suggestion
        image: "{{ .Values.image }}:{{ default .Chart.AppVersion  .Values.imageTag }}"
```

---

I'm not sure if "correct" is the best description. The [documentation](https://helm.sh/docs/chart_template_guide/functions_and_pipelines/#using-the-default-function) uses the pipeline syntax, and also says:
> Inverting the order is a common practice in templates. You will see `.val | quote` more often than `quote .val`. Either practice is fine.

That said, the logic is the same either way so happy to make this change if you really want.